packages/shared/src/utils/platform/platform.js

```diff
@@ -12,18 +12,22 @@ export const isMT5 = () =>
     /^\/mt5/.test(window.location.pathname) ||
     (/^\/(br_)/.test(window.location.pathname) && window.location.pathname.split('/')[2] === 'mt5');
 
-export const getPlatformHeader = routing_history => {
-    if (isBot() || isNavigationFromPlatform(routing_history, routes.bot)) return 'DBot';
-    if (isMT5() || isNavigationFromPlatform(routing_history, routes.mt5)) return 'DMT5';
-    if (isNavigationFromPlatform(routing_history, routes.smarttrader)) return 'SmartTrader';
-    return 'DTrader';
-};
+export const isTrader = () =>
+    (/^\//.test(window.location.pathname) && window.location.pathname.length === 1) ||
+    (/^\/(br_)/.test(window.location.pathname) && window.location.pathname.split('/')[2] === '');
+
+// TODO: Replace platform names with `platform_name` enum after merging "carol/journal error notification"
+export const getPlatformInformation = routing_history => {
+    const should_be_bot = isBot() || isNavigationFromPlatform(routing_history, routes.bot);
+    if (should_be_bot) return { header: 'DBot', icon: 'IcBrandDbot' };
+
+    const should_be_mt5 = isMT5() || isNavigationFromPlatform(routing_history, routes.mt5);
+    if (should_be_mt5) return { header: 'DMT5', icon: 'IcBrandDmt5' };
+
+    const should_be_smartTrader = isNavigationFromPlatform(routing_history, routes.smarttrader);
+    if (should_be_smartTrader) return { header: 'SmartTrader', icon: 'IcBrandSmarttrader' };
```

Comment:
```suggestion
export const getPlatformInformation = routing_history => {
    if (isBot() || isNavigationFromPlatform(routing_history, routes.bot)) return { header: 'DBot', icon: 'IcBrandDbot' };

    if (isMT5() || isNavigationFromPlatform(routing_history, routes.mt5)) return { header: 'DMT5', icon: 'IcBrandDmt5' };

    if ( isNavigationFromPlatform(routing_history, routes.smarttrader);) return { header: 'SmartTrader', icon: 'IcBrandSmarttrader' };
```

This looks better to me, we don't need the extra `should_be_bot`, `should_be_mt5` and `should_be_smartTrader` then. What do you think?

Reply:
Since the condition was big, I tried to make it to a more readable and understandable condition. It added some lines of code but at least it's more understandable now. And I chose the name base on the goal of the condition.

Reply:
Since we're giving the same conditions to the variable and the conditions aren't complicated, it's better to give it directly to the `if` statement. IMO it's better to use the condition **directly** in the `if statement` because creating **_extra_** variables for these conditions unnecessarily complicates the logic. 🙂